book/src/dev/rfcs/0005-state-updates.md

```diff
@@ -61,7 +62,70 @@ state service.
 # Guide-level explanation
 [guide-level-explanation]: #guide-level-explanation
 
-XXX fill in after writing other details
+The `zebra-state` crate provides an implementation of the chain state storage
+logic in a zcash consensus node. Its main responsibility is to store chain
+state, validating new blocks against the existing chain state in the process,
+and to allow later querying of said chain state. `zebra-state` provides this
+interface via a `tower::Service` based on the actor model with a
+request/response interface for passing messages back and forth between the
+state service and the rest of the application.
+
+The main entry point for the `zebra-state` crate is the `init` function. This
+function takes a `zebra_state::Config` and constructs a new state service,
+which it returns wrapped by a tower::Buffer. This service is then interacted
+with via the `tower::Service` trait.
+
+```rust
+use tower::{Service, ServiceExt};
+
+let config = app_config();
+let state_config = config.state;
+let network = config.network;
```

Comment:
Slight preference not to include `app_config` or other abscissa-inherited details here.  Maybe the example could just create `state`? I'm not sure that all of the config prelude is really helpful to the example, since no user of the crate is going to use it that way anyways.